spec/models/manageiq/providers/hawkular/middleware_manager/alert_manager_spec.rb

```diff
@@ -82,89 +82,96 @@
     end
 
     it '#convert_to_group_conditions Sessions' do
-      MW_WEB_SESSIONS = %w(
-        mw_aggregated_active_web_sessions
-        mw_aggregated_expired_web_sessions
-        mw_aggregated_rejected_web_sessions
-      ).freeze
+      MW_WEB_SESSIONS = [
+        'mw_aggregated_active_web_sessions',   'WildFly Aggregated Web Metrics~Aggregated Active Web Sessions',
```

Comment:
I'm not sure of this.
Native metric names should be in the configuration file loaded, and in theory it shouldn't be manipulated here.
In the next iteration hMetrics will change to a Prometheus and this names will be completely different.
So, it should not hit the MiQ code.
I guess that in there was a utility class to fetch the native metric from the miq name that could be used.
Or perhaps I am missing something.

Reply:
I need it for test that method is getting the right definition because this was the problem that I have. How could I solve it?

Reply:
the mixing should load this info from the config files
https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/live_metrics_mixin.rb
so, it should be available at resource level AFAIR